project/design.md

```diff
@@ -0,0 +1,25 @@
+This is a strawman proposal for the design of Merritt.
+
+# Goal
+The goal of `merritt/merritt` is to create an open source container management application.  This will be assembled from a variety of "extensions" that are implemented in containers.  Each service will provide a standard interface for implementation and a GRPC API for other services to use.
+
+# Components
```

Comment:
No I think they would be standalone that are pulled in as desired.  It makes it easier to use other components with each other without bringing in a bunch of other dependencies.